vip-dashboard.php

```diff
@@ -2,11 +2,11 @@
 
 /*
  * Plugin Name: VIP Dashboard
- * Plugin URI: https://wpvip.com
+ * Plugin URI: http://wpvip.com
  * Description: WordPress VIP Go Dashboard
- * Author: Scott Evans, Filipe Varela, WPVIP
- * Version: 2.1.1
- * Author URI: https://wpvip.com
+ * Author: Scott Evans, Filipe Varela, Pau Argelaguet
+ * Version: 3.0.0
+ * Author URI: http://wpvip.com
```

Comment:
Is there a particular reason for changing it to http?

Reply:
I was copy pasting. Indeed, it is better to have it with `https`.